### PR TITLE
Omnibus marmalade ns, sale into standard, fungible-v2 test coverage and other fixes

### PR DESCRIPTION
**Pull request by sirlensalot**
- move to `marmalade` namespace, `ledger` module name
- add sale pact and caps to `poly-fungible-v2`
- crosschain policy support
- move token policy to `kip`
- standardize policy naming
- add fungible-v2 test coverage to `ledger`
- move all policy enforcements out of `defcap`s into function bodies
- move guard policy init into `create-token`/`enforce-init`

### CODE REVIEW

File: pact/fixed-quote-policy.pact [hunk 1]
```diff
@@ -64,41 +53,43 @@
...
   (defun enforce-init:bool
-    ( token:object{token-policy-v1_DRAFT1.token-info}
+    ( token:object{token-info}
     )
-    (get-policy token)
-    true
+    (insert policies (at 'id token)
+      { 'mint-guard: (read-keyset 'mint-guard)
+      , 'max-supply: (read-decimal 'max-supply)
+      , 'min-amount: (read-decimal 'min-amount) })
```
Comment (ggobugi27) [inline pact/fixed-quote-policy.pact:69]: Add `true`

File: pact/test/fungible.repl [hunk 2]
```diff
@@ -0,0 +1,787 @@
+
```
Comment (ggobugi27) [inline pact/test/fungible.repl:1]: `(load "../root/fungible-v2.pact")`

File: .github/workflows/pact.yml [hunk 3]
```diff
@@ -40,7 +40,7 @@ jobs:
           distribution: "ubuntu-18.04"
       - name: pact/hft.repl
```
Comment (ggobugi27) [inline .github/workflows/pact.yml:41]: Should we add `fungible.repl` in here as well?

File: .github/workflows/pact.yml [hunk 4]
```diff
@@ -40,7 +40,7 @@ jobs:
...
         run: |
-          bin/pact -t pact/hft.repl > out.log 2>&1
+          bin/pact -t pact/marmalade.repl > out.log 2>&1
```
Comment (ggobugi27) [inline .github/workflows/pact.yml:43]: Let's add `fungible.repl` as well?

**Review by ggobugi27 — APPROVED** [review]
The changes look great. Looks good to merge!

**Review by buckie — APPROVED** [review]
Looks good, doesn't seem to impact the frontend

**Comment by ggobugi27** [comment]
> Looks good, doesn't seem to impact the frontend

I think frontend code will need to update its pact codes (remove `init-...` when `create-token`, and add params in env-data)
Also, renaming hft-> marmalade once contract is redeployed.

**Review by jmcardon — APPROVED** [review]

**Pull request merged**